modules/coreos-layering-configuring.adoc

```diff
@@ -44,6 +44,8 @@ RUN rpm-ostree override replace https://example.com/hotfixes/haproxy-1.0.16-5.el
 Instructions on how to create a Containerfile are beyond the scope of this documentation.
 ====
 
+* Because the process for building a custom layered image is performed outside of the cluster, you need to make sure the `pull-secret.txt` file, which is required to pull {product-title} images from quay.io, is available to your tools. You can put the pull secret file in one of the default locations, `~/.config/containers/auth.json`, `~/.docker/config.json`, or  `~/.dockercfgin`. Or, use a random location and point to file by using the `--authfile /path/to/pull-secret.txt` flag with your pull command. 
```

Comment:
How about something like, "....use the `--authfile /path/to/pull-secret` option with podman or buildah. You may also add the pull secret to one of the default file locations to have it read by these tools automatically. Refer to the `containers-auth.json` man page for details. "

For context, this file comes from Step 5 of https://docs.openshift.com/container-platform/4.12/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-obtaining-installer_installing-platform-agnostic
> Download your installation [pull secret from the Red Hat OpenShift Cluster Manager](https://console.redhat.com/openshift/install/pull-secret). This pull secret allows you to authenticate with the services that are provided by the included authorities, including Quay.io, which serves the container images for OpenShift Container Platform components.